src/ngraph/op/batch_norm.hpp

```diff
@@ -93,6 +95,16 @@ namespace ngraph
                                            const NodeVector& deltas) override;
 
         private:
+            enum
```

Comment:
I think I'm worried more about code duplication for BatchNormBackprop rather than keeping this `enum` private

Reply:
There's no guarantee the backprop args are in the same order as the fprop.